### PR TITLE
Add Indonesia translation of readme.md

### PR DESCRIPTION
**Pull request by xTunqki**

### CODE REVIEW

**Comment by Shpota** [comment]
Hi @xTunqki!

Thank you for your contribution!

May I ask you to adjust the translation according to [the README in this branch](https://github.com/Shpota/goxygen/tree/add-postgres)?

We're planning to release a new version in with support of different databases and that will require changing the documentation as well.

There are only a couple of sentences changed.

**Comment by xTunqki** [comment]
Sorry this is my first contribution on github, i don't know how to do it

**Comment by Shpota** [comment]
No worries, you've done everything alright.

File: .github/README_id.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```
[the remark this answers is not in the document]
Comment (xTunqki) [reply .github/README_id.md:3]: why it's 25px? i copied that attribute value from the japan one

File: .github/README_id.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    ├── .gitignore
+    └── README.md                # panduan tentang cara menggunakan repo yang dihasilkan
+
+File seperti unit test atau komponen sampel tidak termasuk di sini
+untuk kesederhanaan.
+
+## Dependensi
+
+Goxygen menghasilkan struktur dasar suatu proyek dan tidak memaksa Anda
+untuk menggunakan seperangkat alat tertentu. Itu sebabnya Goxygen tidak membawa dependensi yang tidak dibutuhkan pada proyek Anda. Hanya menggunakan
```
[the remark this answers is not in the document]
Comment (xTunqki) [reply .github/README_id.md:120]: That's good, but i think "pada" is more suitable in my native language, anyway do you can speak indonesian?

File: .github/README_id.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shpota) [reply .github/README_id.md:3]: It is 25px only for the selected language and 20px for the others. When you click on a particular flag it gets a bit larger - that's what 25px needed for. In this case, Indonesian flag should be larger (as it is selected).

File: .github/README_id.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Shpota) [reply .github/README_id.md:120]: @xTunqki no, she is not. She was just proofreading and checking for typos. If the original version is correct let's keep it.

**Pull request merged**